src/utils.ts

```diff
@@ -283,6 +291,7 @@ export default abstract class BaseCommand extends Command {
     return sequelize
   }
 
+  // TODO: Rework this with correct types, supporting the actual Flags
```

Comment:
I don't see a way how it can work with current flags, because flag object always has type `option` and we can not prepare a condition for prompiting